controller/script/simulate-proxy/main.go

```diff
@@ -22,7 +22,99 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
-/* A simple script for posting simulated telemetry data to the proxy api */
+/* A simple script for exposing simulated prometheus metrics */
+
+type simulatedProxy struct {
+	sleep time.Duration
+}
+
+type proxyVectors struct {
+	requestTotals      *prom.CounterVec
+	responseTotals     *prom.CounterVec
+	responseDurationMs *prom.HistogramVec
+	responseLatencyMs  *prom.HistogramVec
+}
+
+// generateProxyTraffic randomly creates metrics under the guise of a single conduit proxy routing traffic.
+func (s *simulatedProxy) generateProxyTraffic(namespace string, podList []*v1.Pod, proxyVec *proxyVectors) {
+	inboundPod := podList[rand.Intn(len(podList))]
```

Comment:
you may want to make `inboundPod` part of `simulatedProxy`, and then ensure we don't select the same random pod across multiple `simulatedProxy`'s.

taking this a step further, as this is only used to generate this proxy's `deployment`, consider doing `strings.Split(inboundPod.GetName(), "-")[0]` when you create a `simulatedProxy`. also, may be more correct to rename `inboundPod` to `podName` or something, as any pod can be inbound or outbound.